api/_types/nodes._common.d.ts

```diff
@@ -20,5 +20,5 @@ export type NodesResponseBase = {
   _nodes?: Common.NodeStatistics;
 }
 
-export type SampleType = 'block' | 'cpu' | 'wait'
+export type SampleType = string
```

Comment:
This changed from an `enum` to a `oneOf` of `const` in the spec, I think we should be able to handle this pretty easily.

Reply:
Sorry I missed your review.

I'm certain that the codegen can handle oneOf with const. for example [WaitForEvents](https://github.com/opensearch-project/opensearch-api-specification/blob/bddc88aedae99e8497ba85b08926e19e87a33ef0/spec/schemas/_common.yaml#L1258-L1278) is [rendered](https://github.com/opensearch-project/opensearch-js/blob/b85602058649d94fcde6ffc8d87983313b139943/api/_types/_common.d.ts#L898C13-L898C26) as 
`export type WaitForEvents = 'immediate' | 'urgent' | 'high' | 'normal' | 'low' | 'languid'`

This is caused by having `type: string` along with `oneOf` https://github.com/opensearch-project/opensearch-api-specification/blob/bddc88aedae99e8497ba85b08926e19e87a33ef0/spec/schemas/nodes._common.yaml#L18. Looks like OpenAPI does allow this.